frontend/amundsen_application/static/js/pages/TableDetailPage/ReportTableIssue/index.tsx

```diff
@@ -72,16 +74,20 @@ export class ReportTableIssue extends React.Component<
 
   getCreateIssuePayload = (formData: FormData): CreateIssuePayload => {
     const {
+      tableKey,
       tableMetadata: { cluster, database, schema, name },
     } = this.props;
+    const { issuePriority } = this.state;
     const title = formData.get('title') as string;
     const description = formData.get('description') as string;
+    const priorityLevel = issuePriority;
```

Comment:
Nit: Why the new temporal variable? Maybe we can avoid it.
If not, you can rename it directly when you do the destructuring of the state.